src/extension.ts

```diff
@@ -26,6 +26,21 @@ function makeTerminal(title : string) {
     terminal.show();
 }
 
+async function waitForTerminal(title : string) {
+    makeTerminal("LittleFS Upload");
```

Comment:
I think you wanted  `makeTerminal(title)`, no ?

Reply:
Indeed. Thanks for catching that.